app/views/releases/show.html.erb

```diff
@@ -16,6 +16,13 @@
   <label>Created</label>
   <%= relative_time(@release.created_at) %>
 
+  <legend>Commit Status</legend>
+  <p>
+    <% @release.github_status.statuses.each do |status| %>
```

Comment:
inside of `github_ok` too ?

Reply:
I don't think this page is critical to the workflow... let's see if it becomes a problem.